Mathlib/Combinatorics/SimpleGraph/Triangle/Basic.lean

```diff
@@ -18,14 +18,13 @@ This module defines and proves properties about triangles in simple graphs.
 
 ## Main declarations
 
-* `SimpleGraph.farFromTriangleFree`: Predicate for a graph to have enough triangles that, to
-  remove all of them, one must one must remove a lot of edges. This is the crux of the Triangle
-  Removal lemma.
+* `SimpleGraph.farFromTriangleFree`: Predicate for a graph to have enough triangles so that
+  a lot of edges must be removed to remove all triangles.
+  This is the crux of the Triangle Removal Lemma.
```

Comment:
@b-mehta, do you think this reads better or worse? To me this reads worse.

Reply:
I think it reads slightly worse, but note there is also a typo in the original: "one must one must".

Reply:
Here's some wordsmithing?
```
* `SimpleGraph.FarFromTriangleFree`: Predicate for a graph such that one must remove a lot of edges
  from it for it to become triangle-free.
  This is the crux of the Triangle Removal Lemma.
```

Reply:
I like that!

Reply:
Done.